### ISSUE TITLE
Importing styles in dynamic component causes error

### ISSUE THREAD

**Issue by shauravg** [zeit/next.js#4717]
# Bug report

## Describe the bug
I have a page search.js which includes a component ListingResults.js via dynamic import and ssr: false. ListingResults.js imports two other components which import their own styles. In dev mode, everything works fine. In production mode, ListingResults.js gets stuck in the "loading..." and doesn't produce any errors on the console. 

After experimenting quite a bit, I could get the components to load properly in production mode if I commented out the styles import line. 
//import styles from "./styles.scss";

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:
1. Import styles in a component that is imported via dynamic import with ssr: false
2. Load the page 
3. Note the component stays in the "loading..." phase

## Expected behavior
Components should load fine. 

## System information
 - OS: macOS
 - Browser: Chrome 65
 - Version of Next.js: 6.0.0
- next-css: 0.2.0, next-sass: 0.1.1, node-sass: 4.7.2

## Additional context

next.config.js:

```
const withSass = require('@zeit/next-sass');
const withCSS = require('@zeit/next-css');
const commonsChunkConfig = require('@zeit/next-css/commons-chunk-config');
module.exports = withCSS(withSass({
  webpack: (config, {isServer}) => {
    // Fixes npm packages that depend on `fs` module
    config.node = {
      fs: 'empty'
    }
    if(!isServer)
    	config = commonsChunkConfig(config, /\.(sass|scss|css)$/)

    return config
  },
  useFileSystemPublicRoutes: false,
}))

```

**Comment by stramel** [comment]
Based on the limited information, I would guess that it is related to one of the many dynamic import issues that Next currently has. #3775 seems to be the parent one and #4639 will hopefully address this.

**Comment by shauravg** [comment]
Hi @stramel What other additional details should I provide?

**Comment by timneutkens** [comment]
This is definitely going to be fixed after #4639 lands, but could you copy-paste this issue to github.com/zeit/next-plugins, so that I'll see it again once I'm working on that. Thanks 🙏
